src/GraphQL.ApiTests/netstandard20+netstandard21/GraphQL.approved.txt

```diff
@@ -409,6 +417,12 @@ namespace GraphQL
         public static T ToObject<T>(this System.Collections.Generic.IDictionary<string, object?> source)
             where T :  class { }
     }
+    [System.AttributeUsage(System.AttributeTargets.Class | System.AttributeTargets.All)]
+    public class OutputTypeAttribute : System.Attribute
```

Comment:
@Shane32 How do you like api approval experience now?

Reply:
It’s working well. 👍 I simply ignore one of the two approval files during any review but it’s nice that it auto updates.